### PR TITLE
[Review] Request from '' @ 'yast/ycp-killer/last-minute-readme-fixes'

### PR DESCRIPTION
**Pull request by dmajda**
Please review the following changes:
- 00e1666 README.md: Add info about YaST module support & qualify few statements
- 064b82d README.md: Link the "Module metadata" section better
- 6840493 README.md: Improve workflow description in the "Overview" section
- 4be874a README.md: Fix small errors
- 263758e README.md: Add packages needed for testing to installation instructions
- 39a489e README.md: Use backslashes to split a long command
- 66ec073 README.md: Improve text in the "Update ycpc" section
- 3aa1a3a README.md: Improve intro text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,25 +183,26 @@ Overview
 --------
 
 YCP Killer is a command-line tool built around tasks that are applied on YaST
-module source code. Some of these tasks are driven by *module metadata files*
-which contain various information about all the translated modules (see [Module
-Metadata](#module-metadata).
+module source code. Some of these tasks are driven by *[module metadata
+files](#module-metadata)* which contain various information about all the
+translated modules. In general, YCP Killer supports only YaST modules for which
+is has metadata available.
```
Comment (lslezak) [inline README.md:189]: A typo: **is**, should be **it**

File: README.md [hunk 2]
```diff
@@ -161,25 +183,26 @@ Overview
...
   * Compile YaST module's YCP modules (without this any code depending on them
     can't be translated by Y2R).
-  * Convert the YaST module source code into Ruby.
+  * Convert YaST module YCP files into Ruby.
   * Generate `Makefile.am` files in all source directories matching the new
     structure (usually only `src`).
   * Create a YaST module package source.
   * Build the package locally.
-  * Submit a package into OBS.
+  * Submit the package source to OBS.
```
Comment (lslezak) [inline README.md:205]: "the source package" ?

**Comment by lslezak** [comment]
LGTM (after fixing the typos)

File: README.md (hunk 1 above)
Comment (dmajda) [inline README.md:189]: Fixed in #608, thanks for catching it.

File: README.md (hunk 2 above)
Comment (dmajda) [inline README.md:205]: No, "package source" is correct. It' a shortcut for "package source code", which means the thing checked-out/commited to OBS. I use this term consistently in several places in the README.

**Comment by dmajda** [comment]
Obsoleted by #608.

File: README.md (hunk 2 above)
Comment (lslezak) [inline README.md:205]: OK, thanks for the explanation.

**Pull request closed without merging**